backend/config/corpora-api.yml

```diff
@@ -630,6 +667,30 @@ components:
           type: string
         detail:
           type: string
+    gene_sets:
+      type: object
+      description: User created geneset information
+      properties:
+        gene_set_name:
+          type: string
+        gene_set_description:
+          type: string
+        genes:
+          type: array
+          items:
+            type: object
+            description: Gene specific information
+            properties:
+              gene_symbol:
+                type:
+                  string
```

Comment:
nit: why is this on a different line? Same question for the properties bellow.

Reply:
no idea... updating